src/applications/mhv/medications/containers/PrescriptionDetails.jsx

```diff
@@ -349,6 +366,9 @@ const PrescriptionDetails = () => {
       generatePDF();
     } else if (pdfTxtGenerateStatus.format === DOWNLOAD_FORMAT.TXT) {
       generateTXT(buildAllergiesTXT());
+    } else {
+      setPdfTxtGenerateStatus({ status: PDF_TXT_GENERATE_STATUS.NotStarted });
```

Comment:
do we need to set status here? i.e. two previous conditions are not doing it.

Reply:
@vmenshutin-bylight Yeah, we should do it. The above conditions are doing it in `generatePDF` and `generateTXT` function.